[2021-03-03 19:29:25,736] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: send_email.send_email 2021-03-03T19:29:21.860178+00:00 [queued]>
[2021-03-03 19:29:25,745] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: send_email.send_email 2021-03-03T19:29:21.860178+00:00 [queued]>
[2021-03-03 19:29:25,745] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-03 19:29:25,745] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-03-03 19:29:25,745] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-03 19:29:25,751] {taskinstance.py:1063} INFO - Executing <Task(EmailOperator): send_email> on 2021-03-03T19:29:21.860178+00:00
[2021-03-03 19:29:25,754] {standard_task_runner.py:52} INFO - Started process 50 to run task
[2021-03-03 19:29:25,756] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'send_email', 'send_email', '2021-03-03T19:29:21.860178+00:00', '--job-id', '31', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/send_email.py', '--cfg-path', '/tmp/tmp2on9zvqa', '--error-file', '/tmp/tmp863ns5q6']
[2021-03-03 19:29:25,756] {standard_task_runner.py:77} INFO - Job 31: Subtask send_email
[2021-03-03 19:29:25,782] {logging_mixin.py:104} INFO - Running <TaskInstance: send_email.send_email 2021-03-03T19:29:21.860178+00:00 [running]> on host 4c61e128b408
[2021-03-03 19:29:25,813] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=iotoi
AIRFLOW_CTX_DAG_ID=send_email
AIRFLOW_CTX_TASK_ID=send_email
AIRFLOW_CTX_EXECUTION_DATE=2021-03-03T19:29:21.860178+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-03-03T19:29:21.860178+00:00
[2021-03-03 19:29:25,839] {logging_mixin.py:104} WARNING - /home/airflow/.local/lib/python3.6/site-packages/airflow/providers/sendgrid/utils/emailer.py:122 PendingDeprecationWarning: Fetching Sendgrid credentials from environment variables will be deprecated in a future release. Please set credentials using a connection instead.
[2021-03-03 19:29:26,158] {emailer.py:147} INFO - Email with subject Thank you for a dinner :-* is successfully sent to recipients: [{'to': [{'email': 'lukas.kellerstein@gmail.com'}]}]
[2021-03-03 19:29:26,166] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=send_email, task_id=send_email, execution_date=20210303T192921, start_date=20210303T192925, end_date=20210303T192926
[2021-03-03 19:29:26,182] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-03-03 19:29:26,210] {local_task_job.py:146} INFO - Task exited with return code 0
